site/docs/master/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+Use this doc to kick the tires and get started with Velero.
+Refer [this document](advanced-installation.md) for more advanced install instructions
+
+## Prerequisites
+
+- access to a Kubernetes cluster, v1.10 or later, with DNS and container networking enabled.
+- `kubectl` installed locally
+
+Velero uses object storage to store backups and associated artifacts. It also optionally integrates with supported block storage systems to snapshot your persistent volumes. Before beginning the installation process, you should identify the object storage provider and optional block storage provider(s) you'll be using from the list of [compatible providers][0]. 
+
+There are supported storage providers for both cloud-provider environments and on-premises environments. For more details on on-premises scenarios, see the [on-premises documentation][2].
+
+## Install the CLI
+
+### Option 1: macOS - Homebrew
+
+On macOS, you can use [Homebrew](https://brew.sh) to install the `velero` client:
+
+```bash
+brew install velero
+```
+
+### Option 2: GitHub release
+
+1. Download the [latest release][1]'s tarball for your client platform.
+1. Extract the tarball:
+
+   ```bash
+   tar -xvf <RELEASE-TARBALL-NAME>.tar.gz
+   ```
+
+1. Move the extracted `velero` binary to somewhere in your `$PATH` (e.g. `/usr/local/bin` for most users).
+
+## Install and configure the server components
+
+There are two supported methods for installing the Velero server components:
+
+- the `velero install` CLI command
+- the [Helm chart](https://github.com/helm/charts/tree/master/stable/velero)
+
+To install and configure the Velero server components, follow the provider-specific instructions documented by [your storage provider][0].
```

Comment:
We got some feedback that this line is unclear (see https://github.com/vmware-tanzu/velero/issues/1910#issuecomment-553543870), since it just links you to the supported-providers page which doesn't have a column titled "isntall instructions" or anything like that - kind of leaves the user to figure out how to navigate through. It'd be great if we could clarify that as well.

Reply:
We might want to add a few more words indicating that the specific steps/options for installing the server components vary depending on which storage provider you're using, so to find the provider-specific instructions, you need to go to your storage provider's plugin docs, links to which can be found on the "supported-providers" page.

Reply:
I've updated this. ptal. Feel free to suggest copy-edits.